File-Formats/PDF/Working-with-Security.md

```diff
@@ -1336,3 +1336,90 @@ Returns user password</td><td>
+    // Load the PDF document from the FileStream with the specified owner password.
+    PdfLoadedDocument loadedDocument = new PdfLoadedDocument(fileStream, "12345");
+    // Access the security settings of the loaded PDF document.
+    PdfSecurity security = loadedDocument.Security;
+    // Disable incremental update for the file structure.
+    loadedDocument.FileStructure.IncrementalUpdate = false;
+    // Set specific permissions for the PDF document.
+    loadedDocument.Security.Permissions = PdfPermissionsFlags.EditAnnotations | PdfPermissionsFlags.AssembleDocument;
+    // Set the encryption algorithm to AES for the document security.
+    security.Algorithm = PdfEncryptionAlgorithm.AES;
+    // Set the owner password for the PDF document.
+    security.OwnerPassword = "";
+    // Set the user password for the PDF document.
+    security.UserPassword = "";
+    // Create a MemoryStream to store the modified PDF document.
+    MemoryStream stream = new MemoryStream();
+    // Save the modified PDF document to the MemoryStream.
+    loadedDocument.Save(stream);
+    // Write the contents of the MemoryStream.
+    File.WriteAllBytes("output.pdf", stream.ToArray());
+    // Close the loaded PDF document.
+    loadedDocument.Close(true);
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+
+    // Load the PDF document "OwnerPasswordOnly.pdf" with the owner password.
+    PdfLoadedDocument loadedDocument = new PdfLoadedDocument("OwnerPasswordOnly.pdf", "12345");
+    // Access the security settings of the loaded PDF document.
+    PdfSecurity security = loadedDocument.Security;
+    // Disable incremental updates for the file structure.
+    loadedDocument.FileStructure.IncrementalUpdate = false;
+    // Set specific permissions for the PDF document, allowing editing annotations and assembling the document.
+    loadedDocument.Security.Permissions = PdfPermissionsFlags.EditAnnotations | PdfPermissionsFlags.AssembleDocument;
+    // Set the encryption algorithm to AES for enhanced security.
+    security.Algorithm = PdfEncryptionAlgorithm.AES;
+    // Set empty owner password.
+    security.OwnerPassword = "";
+    // Set empty user password.
+    security.UserPassword = "";
+    // Save the modified document.
+    loadedDocument.Save("output.pdf");
+    // Close the document.
+    loadedDocument.Close(true);
+
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+
+    ' Load the PDF document "OwnerPasswordOnly.pdf" with the owner password "12345"
+    Dim loadedDocument As New PdfLoadedDocument("OwnerPasswordOnly.pdf", "12345")
+    ' Access the security settings of the loaded PDF document
+    Dim security As PdfSecurity = loadedDocument.Security
+    ' Disable incremental updates for the file structure
+    loadedDocument.FileStructure.IncrementalUpdate = False
+    ' Set specific permissions for the PDF document, allowing editing annotations and assembling the document
```

Comment:
' Set specific permissions for a PDF document, allowing editing annotations and assembling the document.